### PR TITLE
[CircleEditor] Add class sidebar.EditAttributesView

### PR DESCRIPTION
**Pull request by rudalsd**
This PR adds class sidebar.EditAttributesView.

ONE-vscode-DCO-1.0-Signed-off-by: Gyeongmin Kwon rudalsd@naver.com

### CODE REVIEW

File: media/CircleEditor/view-sidebar.js [hunk 1]
```diff
@@ -273,24 +252,7 @@ sidebar.NodeSidebar = class {
...
-        if (this._isCustom === true) {
-            this._attributesBox = this._host.document.createElement('div');
-            this._attributesBox.appendChild(view.render());
-            this._elements.push(this._attributesBox);
-        } else {
-            this._elements.push(view.render());
-        }
-    }
-
+
```
Comment (llFreetimell) [inline media/CircleEditor/view-sidebar.js:255]:
```suggestion

```

**Comment by llFreetimell** [comment]
Indentaion used 4 blanks instead of 2 blanks T.T

**Comment by rudalsd** [comment]
Shoud I change indentation to 2 blanks?

**Comment by llFreetimell** [comment]
I forgot that clang-format requires 4 blanks... sorry for confusion :(

**Review by YuyeonHan — APPROVED** [review]
LGTM

**Pull request merged**